src/API/OnboardingProfile.php

```diff
@@ -240,12 +240,7 @@ public static function get_profile_properties() {
 				'description'       => __( 'Industry.', 'woocommerce-admin' ),
 				'context'           => array( 'view' ),
 				'readonly'          => true,
-				'sanitize_callback' => 'wp_parse_slug_list',
 				'validate_callback' => 'rest_validate_request_arg',
-				'items'             => array(
-					'enum' => array_keys( Onboarding::get_allowed_industries() ),
-					'type' => 'string',
-				),
```

Comment:
Is it possible to describe the new structure in JSON Schema here instead of removing validation?

Reply:
Good idea! I'll do it. I'll validate the datatype: `json`.

Reply:
Interesting.. it seems this isn't used anyhow: https://github.com/woocommerce/woocommerce-admin/blame/b75ee0cbccb0540e69ff7c3b29f6b4f3f8b2039d/src/API/OnboardingProfile.php#L370-L376

Ideally we'd define the full schema, not just "type = json", but since the validation isn't being used we can save that for a future PR. _cc: @joshuatf who wrote the referenced code_